ts/Drawer/index.tsx

```diff
@@ -1,5 +1,5 @@
-import {MDCPersistentDrawer} from '@material/drawer/persistent';
-import {MDCTemporaryDrawer} from '@material/drawer/temporary';
+// @ts-ignore
```

Comment:
Are the updates to the TypeScript definitions?

Reply:
@type did not update after 0.35 i am in favor of removing it

Reply:
Yes I think until it updates (if it ever does) we should remove it

Reply:
Then please remove it and then remove that `@ts-ignore`